python/pythonmonkey/builtin_modules/timers.js

```diff
@@ -53,6 +53,10 @@ function setTimeout(handler, delayMs = 0, ...args)
  */
 function clearTimeout(timeoutId) 
 {
+  // silently does nothing when an invalid timeoutId (should be an int32 value) is passed in
+  if (!Number.isInteger(timeoutId))
```

Comment:
Is this not now incompatible with socket.io as described in https://github.com/Distributive-Network/PythonMonkey/issues/232#issuecomment-1935063360

Reply:
No. JS numbers are always float64/`double` values **in the JS context**. [`Number.isInteger()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) only detects if the value is integral, regardless of whether it's stored as int32 or double internally.